activerecord/test/cases/associations/belongs_to_associations_test.rb

```diff
@@ -1096,6 +1099,27 @@ def test_association_force_reload_with_only_true_is_deprecated
 
     assert_deprecated { client.firm(true) }
   end
+
+  #  Next two tests, please see https://github.com/rails/rails/issues/20602
```

Comment:
Probably best to leave this to the commit message - we can check the git history to see where and why the tests were added.